### ISSUE TITLE
loading observable

### ISSUE THREAD

**Issue by Charl---** [micouz/sparix#1]
What about a loading$ property on Store class that emit a boolean ?

**Comment by micouz** [comment]
Yes, that would be great for Stores that are specialized in storing data
fetched asynchronously from the server (as most are, obviously). However,
the sparix Store can also be used for synchronous data (a shopping cart,
for example), in which case the loading$ property wouldn't make much sense.

It may be time to start working on an HttpStore or RESTfulStore or
BackendStore (or a separate sparix-data project) to provide generic CRUD
functionnality, among other things. In the meantime, it is pretty
straightforward to add a loading$ property to an existing Store, or to a
Store's abstract subclass.

--
Mikael Couzic
Appartement 131
32, rue du Javelot
75013 Paris
06 51 17 97 83


2016-05-23 20:28 GMT+02:00 Charles Jacquin <notifications@github.com>:
